coldfront/core/project/tests/test_commands/test_add_service_units_to_project.py

```diff
@@ -0,0 +1,171 @@
+from io import StringIO
+import sys
+
+from django.core.management import call_command, CommandError
+
+from coldfront.api.allocation.tests.test_allocation_base import TestAllocationBase
+from coldfront.api.statistics.utils import get_accounting_allocation_objects
+from coldfront.core.allocation.models import Allocation, AllocationAttributeType
+from coldfront.core.project.models import Project
+from coldfront.core.resource.models import Resource
+from coldfront.core.statistics.models import ProjectTransaction, ProjectUserTransaction
+from coldfront.core.utils.common import utc_now_offset_aware
+
+
+class TestAddServiceUnitsToProject(TestAllocationBase):
+    """Class for testing the management command add_service_units_to_project"""
+
+    def setUp(self):
+        """Set up test data."""
+        super().setUp()
+
+    def allocation_values_test(self, project, value, user_value):
+        allocation_objects = get_accounting_allocation_objects(project)
+        self.assertEqual(allocation_objects.allocation_attribute.value, value)
+
+        for project_user in project.projectuser_set.all():
+            if project_user.role.name != 'User':
+                continue
+            allocation_objects = get_accounting_allocation_objects(
+                project, user=project_user.user)
+
+            self.assertEqual(allocation_objects.allocation_user_attribute.value,
+                             user_value)
+
+    def transactions_created(self, project, pre_time, post_time):
```

Comment:
This (and `historical_objects_updated`) aren't fully testing that objects were created--they could have existed for whatever reason prior to the command invocation.

We should test that the object count was incremented.